### DISCUSSION TITLE
Force RMSE in multi component system

### DISCUSSION THREAD

**Discussion by ald3hyde** [deepmodeling/deepmd-kit#2663] [Q&A] [answered]
I am currently trying to create a multi-component model of a 5-element system. I have encountered a strange phenomenon: the RMSE behavior for the 3-component system is natural for the supervised data, but the RMSE behavior for the 4- and 5-component systems is off by a certain rule. Could this be caused by AIMD result ?

All data are at high temperatures (3000K).
The data for the plots were generated by dp test.
・5 component system
<img width="402" alt="image" src="https://github.com/deepmodeling/deepmd-kit/assets/123191059/e4e3770b-c7ab-4d85-bd92-33a87dd549d2">
・3 component system
<img width="411" alt="image" src="https://github.com/deepmodeling/deepmd-kit/assets/123191059/8c808dc0-b3cb-4862-9146-e95c9a94dcea">

・my input.json script
    "descriptor": {
      "type": "se_atten",
      "sel": 110,
      "rcut": 6,
      "rcut_smth": 0.5,
      "neuron": [
        25,
        50,
        100
      ],
      "axis_neuron": 16,
      "resnet_dt": false,
      "seed": 0,
      "attn": 120,
      "attn_layer": 0,
      "attn_dotr": true,
      "attn_mask": false
    },
    "fitting_net": {
      "type": "ener",
      "neuron": [
        240,
        240,
        240
      ],
      "resnet_dt": true,
      "seed": 1
    }
  },
  "learning_rate": {
    "type": "exp",
    "start_lr": 0.001,
    "stop_lr": 3.51e-8,
    "decay_steps": 5000
  },
  "loss": {
    "type": "ener",
    "start_pref_e": 0.02,
    "limit_pref_e": 1,
    "start_pref_f": 10000,
    "limit_pref_f": 1,
    "start_pref_v": 0,
    "limit_pref_v": 0,
    "relative_f": 10

**Comment by njzjz** [comment]
Could you show a sample of the data?

**Reply by ald3hyde** [reply]
show you maximum force values at each step
・5-component(2000K)(Extracted every 10steps after 5000 step)
![image](https://github.com/deepmodeling/deepmd-kit/assets/123191059/85cc5904-5ec7-46f0-bf0b-fb0568ce4120)
・3-component(3000K)(Extracted every 10steps after 2000 step)
![image](https://github.com/deepmodeling/deepmd-kit/assets/123191059/39459136-5099-4a57-913d-31b2bcbfb039)

**Reply by njzjz** [reply, marked as answer]
These figures do not contain much information. It will be helpful if you show a structure with a strange error. I guess there is interaction in the range of more than 6 Å in these structures.

**Reply by ald3hyde** [reply]
The target is in a molten salt state. Strangely enough, the structure (RDF) is very good, but when fitting to multi-component (with sodium) data, it ends up being the same. However, when fitting multi-component (with sodium) data, I get the same results. Is this a result of long-range interaction due to sodium?
